lib/net.c

```diff
@@ -1200,12 +1201,102 @@ start_iface(struct gatekeeper_if *iface, unsigned int num_attempts_link_get)
 	return ret;
 }
 
+static void
+calculate_net_config_para(struct net_config *net_conf)
+{
+	int i;
+
+	/* The maximum number of RX queues on any interface. */
+	uint16_t gatekeeper_num_rx_queues = net_conf->back_iface_enabled ?
+		RTE_MAX(net_conf->front.num_rx_queues,
```

Comment:
1. In fact, we don't want the maximum, but the sum! The new value should be as follows:
`calculate_net_config_para + (net_conf->back_iface_enabled ? net_conf->back.num_rx_queues : 0);`

2. Replace the word `maximum` with `total` in the comment.

3. Apply items 1 and 2 above to `gatekeeper_num_tx_queues` below.